public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/OperatorStatusItem.tsx

```diff
@@ -19,21 +22,28 @@ export const OperatorStatusItem: FC<OperatorStatusItemProps> = ({
 }) => {
   const styles = useStyles(getStyles);
 
+  const showStatus = !(
+    kubernetes.status === KubernetesClusterStatus.provisioning && operator.status === Status.unavailable
+  );
+
   return (
     <div className={styles.connectionItemWrapper} data-testid={dataTestId}>
       <span className={styles.connectionItemLabel} title={OPERATOR_FULL_LABELS[databaseType]}>
-        {buildOperatorLabel(operator, databaseType)}:
-      </span>
-      <span className={styles.connectionItemValue}>
-        <KubernetesOperatorStatus
-          operator={operator}
-          databaseType={databaseType}
-          kubernetes={kubernetes}
-          setSelectedCluster={setSelectedCluster}
-          setOperatorToUpdate={setOperatorToUpdate}
-          setUpdateOperatorModalVisible={setUpdateOperatorModalVisible}
-        />
+        {buildOperatorLabel(operator, databaseType)}
+        {`${showStatus ? ':' : ''}`}
```

Comment:
Maybe I'm wrong, but I think we don't need the outermost backticks

Reply:
thanks! you are right)